unlock-app/src/components/interface/connect/ConnectWalletComponent.tsx

```diff
@@ -1,41 +1,15 @@
-import { useAuth } from '~/contexts/AuthenticationContext'
-import { ConnectedWallet } from './ConnectedWallet'
 import { ConnectWallet } from './Wallet'
 import { CheckoutService } from '../checkout/main/checkoutMachine'
-import { useSIWE } from '~/hooks/useSIWE'
-import { useSession } from 'next-auth/react'
-import ConnectingWaas from './ConnectingWaas'
 
 interface ConnectWalletComponentProps {
   onNext?: () => void
   checkoutService?: CheckoutService
-  shoudOpenConnectModal?: boolean
 }
 
-const ConnectWalletComponent = ({
-  onNext,
-  checkoutService,
-  shoudOpenConnectModal = false,
-}: ConnectWalletComponentProps) => {
-  const { account, connected } = useAuth()
-  const { isSignedIn } = useSIWE()
-
-  const { data: session } = useSession()
-  const isLoadingWaas = session && (!connected || !isSignedIn || account === '')
-
-  if (isLoadingWaas) {
-    return <ConnectingWaas />
-  }
-
+const ConnectWalletComponent = () => {
   return (
     <div>
-      {!connected && (
-        <ConnectWallet
-          shoudOpenConnectModal={shoudOpenConnectModal}
-          checkoutService={checkoutService}
-        />
-      )}
-      {connected && <ConnectedWallet onNext={onNext} />}
+      <ConnectWallet />
```

Comment:
We can probably get rid of the whole `ConnectWalletComponent` component, no? It does not add any logic

Reply:
yup we can!